src/redux/request.js

```diff
@@ -11,6 +11,7 @@ import 'rxjs/add/operator/catch';
 export const REQUEST_MAKE = '@@ui-eholdings/REQUEST_MAKE';
 export const REQUEST_RESOLVE = '@@ui-eholdings/REQUEST_RESOLVE';
 export const REQUEST_REJECT = '@@ui-eholdings/REQUEST_REJECT';
+export const REQUEST_CLEAR = '@@ui-eholdings/REQUEST_CLEAR';
```

Comment:
not sure how i feel about the naming here.  the other `REQUEST_` constants deal with http requests, and also set special state in the reducer based on those external interactions.  `REQUEST_CLEAR` doesn't quite fit that definition, since it doesn't make any kind of request and simply manipulates redux state.